packages/db/src/test/networkGenealogy.spec.ts

```diff
@@ -0,0 +1,284 @@
+import {generateId, WorkspaceClient} from "./utils";
+import {
+  FindAncestors,
+  FindDescendants,
+  AddNetworkGenealogies
+} from "./networkGenealogy.graphql";
+import {AddNetworks} from "./network.graphql";
+
+// returnBlockHeader mocks the bare-bones return of a call to web3,
+// to check whether a network resource is a part of the network we're connected to
+const returnBlockHeader = blockNumber => {
```

Comment:
What do you think of the name `findBlockByNumber`? `returnBlockHeader` sounds like a noun, and functions should be verbs :)